xml/System/Array.xml

```diff
@@ -6416,10 +6416,14 @@ int[,,] TDArray = new int[1,1,1];
         <Parameter Name="array" Type="T[]" Index="0" FrameworkAlternate="netcore-2.0;netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <typeparam name="T">To be added.</typeparam>
-        <param name="array">To be added.</param>
-        <summary>To be added.</summary>
+        <typeparam name="T">The type of the elements in <paramref name="array" />.</typeparam>
+        <param name="array">The one-dimensional <see cref="T:System.Array" /> of elements of type <typeparamref name="T" /> to reverse.</param>
+        <summary>Reverses the sequence of the elements in the entire one-dimensional <see cref="T:System.Array" /> of elements of type <typeparamref name="T" />.</summary>
```

Comment:
Does it needs to be that specific?

```suggestion
        <summary>Reverses the sequence of the elements in the one-dimensional array.</summary>
```

Reply:
Same. Need to disambiguate.

Reply:
Then what about disambiguate saying *generic* array:

```suggestion
        <summary>Reverses the sequence of the elements in the one-dimensional generic array.</summary>
```